src/swapd/swap_state.rs

```diff
@@ -2135,10 +2192,34 @@ fn attempt_transition_to_alice_reveal(
     }
 }
 
+fn bob_watch_cancel_output_task(runtime: &mut Runtime) -> Option<Task> {
```

Comment:
Implement this on the runtime directly, like other places where we mutate the Runtime.

Reply:
done in c9da29f628564a787638d76acb6263a7165fb223